### DISCUSSION TITLE
RESOLVED Purchased license, received nothing

### DISCUSSION THREAD

**Discussion by cloudhal** [rustdesk/rustdesk#8575] [General]
RESOLVED - email issue my end sorry! I've paid for a license, and received nothing in my email. Support haven't answered my email. Fair enough don't support OSS users, but paying customers do actually need some support, not to mention the product that they have paid for. We've been stuck a whole day on implementation because you haven't fulfilled your side of the contract.

**Comment by rustdesk** [comment]
I have replied back to you 3 times, the last time I sent mail to you with my personal email, please check. Please double check your spam

**Reply by rustdesk** [reply]
BTW, I just received mail delivery failure, it looks like you block our domain. rustdesk.com

**Reply by cloudhal** [reply]
Hi, no we are not blocking this. Email is (removed)

**Reply by cloudhal** [reply]
Can you paste the NDR please? I've run a trace and checked spam filters, nothing arrived from you guys. Thanks

**Reply by cloudhal** [reply]
All sorted - they all came through, thanks.
